requirements.txt

```diff
@@ -22,17 +29,23 @@ MarkupSafe==1.1.0
 marshmallow==2.16.3
 mccabe==0.6.1
 more-itertools==4.3.0
+passlib==1.7.1
 pathlib2==2.3.3
 pluggy==0.8.0
+psycopg2==2.7.4
 py==1.7.0
+pycodestyle==2.4.0
+pyflakes==2.0.0
+PyJWT==1.4.2
 pylint==1.9.3
-psycopg2==2.7.4
 pytest==4.0.1
 pytest-cov==2.6.0
 python-dotenv==0.10.0
 pytz==2018.7
+requests==2.21.0
 scandir==1.9.0
 singledispatch==3.4.0.3
 six==1.11.0
+urllib3==1.24.1
 Werkzeug==0.14.1
 wrapt==1.10.11
```

Comment:
You have so many files installed. Are you sure you are using all the packages listed here in the requirements.txt?